Aplib.Core.Tests/Intent/Tactics/TacticTests.cs

```diff
@@ -169,6 +169,21 @@ public void AnyOfTacticGetAction_WhenASubtacticIsEnabled_ReturnsActionOfEnabledS
         selectedAction.Should().Be(action1);
     }
 
+    [Fact]
+    public void AnyOfTactic_WithFailingGuard_ReturnsNoAction()
+    {
+        // Arrange
+        Action<IBeliefSet> action1 = new(_ => { });
+        PrimitiveTactic<IBeliefSet> tactic1 = new(action1, _ => true);
```

Comment:
Maybe I'm missing something?
```suggestion
        Action<IBeliefSet> action = new(_ => { });
        PrimitiveTactic<IBeliefSet> tactic = new(action, _ => true);
```